kotest-assertions/src/commonMain/kotlin/io/kotest/properties/Gen.kt

```diff
@@ -46,6 +47,10 @@ interface Gen<T> {
 
   fun shrinker(): Shrinker<T>? = null
 
+  fun uniqueRandoms(seed: Long? = null): Sequence<T> = sequence {
+     yieldAll(random(seed).distinct())
+  }
```

Comment:
Can we make this an extension method on Generator and not defined inside the interface itself. It's fine to keep it in the same file just below the interface definition.

Reply:
Updated